doc/source/cluster/kubernetes/examples/rayjob-kueue-priority-scheduling.md

```diff
@@ -0,0 +1,317 @@
+(kuberay-kueue-priority-scheduling-example)=
+
+# Priority Scheduling with RayJob and Kueue
+
+In this guide, we will run [Fine-tune a PyTorch Lightning Text Classifier with Ray Data](https://docs.ray.io/en/master/train/examples/lightning/lightning_cola_advanced.html) as a RayJob and leverage Kueue to orchestrate priority scheduling and quota management.
+
+## What is Kueue?
+
+[Kueue](https://kueue.sigs.k8s.io/) is a kubernetes-native system that manages quotas
+and how jobs consume them. Kueue decides when a job should wait, when a job should be
+admitted to start (as in pods can be created) and when a job should be preempted (as in
+active pods should be deleted).
+
+Kueue has native support for some of KubeRay's APIs. Specifically, you can use Kueue
+to manage resources consumed by RayJob and RayCluster. See the
+[Kueue documentation](https://kueue.sigs.k8s.io/docs/overview/) to learn more.
+
+## Create a Kubernetes cluster on GKE
+
+If you already have a Kubernetes cluster with GPUs, you can skip this step.
+
+Create a GKE cluster with autoscaling enabled:
+```bash
+gcloud container clusters create kuberay-gpu-cluster \
+    --num-nodes=1 --min-nodes 0 --max-nodes 1 --enable-autoscaling \
+    --zone=us-west1-b --machine-type e2-standard-4 --cluster-version 1.28
+```
+
+Create a GPU node pool:
+```bash
+gcloud container node-pools create gpu-node-pool \
+  --accelerator type=nvidia-l4,count=2,gpu-driver-version=latest \
```

Comment:
We need to remove `gpu-driver-version=latest` and install the NVIDIA driver via [Step 4: Install GPU drivers (optional)](https://anyscale-ray--42903.com.readthedocs.build/en/42903/cluster/kubernetes/user-guides/gcp-gke-gpu-cluster.html#step-4-install-gpu-drivers-optional) instead. See https://github.com/ray-project/ray/pull/42903/files#r1477002737 for more context.